### PR TITLE
Updated README.md and Config.json

### PR DESCRIPTION
**Pull request by Kathund**
README.md - added more info about the config file

Config.json - renamed to **Config-EXAMPLE.json** and edited readme to relect that

### CODE REVIEW

File: config-EXAMPLE.json [hunk 1]
```diff
@@ -28,10 +28,10 @@
     "hypixelAPIkey": "HYPIXEL_API_KEY",
 
     "skyHelperAPI": "http://192.95.42.73:1924//v1",
-    "skyHelperKey": "DuckySoLucky",
+    "skyHelperKey": "DuckySoLucky", // leave this
```
Comment (DuckySoLucky) [inline config-EXAMPLE.json:31]: Comments don't exist in JSON files, so this is gonna cause an error. Also can you please rename file to `config.example.json`

**Review by DuckySoLucky — CHANGES_REQUESTED** [review]
Comments don't exist in JSON files, so this is gonna cause an error. Also can you please rename file to `config.example.json`

**Comment by Kathund** [comment]
@DuckySoLucky Updated and added more info to readme

**Review by DuckySoLucky — APPROVED** [review]
lgtm

**Pull request merged**